test/acceptance_with_go_client/multi_tenancy_tests/activation_deactivation_test.go

```diff
@@ -12,12 +12,13 @@
 package multi_tenancy_tests
 
 import (
-	"acceptance_tests_with_client/fixtures"
 	"context"
 	"fmt"
 	"testing"
 	"time"
 
+	"acceptance_tests_with_client/fixtures"
+
```

Comment:
this line is now marked as not `gofumted` in my workspace, do you use `golangci-lint` tool as a linter?

Reply:
Of course.

> jeremyprice@Jeremys-MacBook-Pro ~/g/weaviate (weaviate-core-refactor)> golangci-lint run                                                                                                                                   (base) 
jeremyprice@Jeremys-MacBook-Pro ~/g/weaviate (weaviate-core-refactor)> git diff                                                                                                                                            (base) 
jeremyprice@Jeremys-MacBook-Pro ~/g/weaviate (weaviate-core-refactor)>

Reply:
```
weaviate/test/acceptance_with_go_client/multi_tenancy_tests (weaviate-core-refactor)$ golangci-lint run
activation_deactivation_test.go:14: File is not `gofumpt`-ed (gofumpt)
import (
activation_deactivation_test.go:19: File is not `gofumpt`-ed (gofumpt)

	"acceptance_tests_with_client/fixtures"
batch_reference_test.go:14: File is not `gofumpt`-ed (gofumpt)
import (
```

Reply:
What version are you using?  It's passing all the github actions tests, and they include gofumpt